CesiumAsync/include/CesiumAsync/SharedAssetDepot.h

```diff
@@ -405,9 +405,13 @@ class CESIUMASYNC_API SharedAssetDepot
       // mutex. So we must take care not to lock it again, which could happen if
       // the asset is currently unreferenced and we naively create an
       // IntrusivePointer for it.
-      pAsset->addReference(true);
+      if (pAsset) {
+        pAsset->addReference(true);
+      }
       CesiumUtility::IntrusivePointer<TAssetType> p = pAsset.get();
-      pAsset->releaseReference(true);
+      if (pAsset) {
```

Comment:
I see how this evolved, but my preference would be for an uninitialized declaration of `p` and then only a single test of `pAsset`.

Reply:
Good idea, done!